### PR TITLE
exception logging

### PR DESCRIPTION
**Pull request by hendrikebbers**
fix for https://github.com/canoo/dolphin-platform/issues/737

---
This change is [Reviewable](https://reviewable.io/reviews/canoo/dolphin-platform/844)

### CODE REVIEW

**Comment by hendrikebbers** [comment]
@blackdrag for you :)

File: platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/ClientSessionExecutorImpl.java [hunk 1]
```diff
@@ -57,6 +61,7 @@ public void run() {
                     T result = task.call();
                     future.complete(result);
                 } catch (Exception e) {
```
Comment (giftkugel) [inline platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/ClientSessionExecutorImpl.java]: Add `final` keyword.

File: platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/ClientSessionExecutorImpl.java [hunk 2]
```diff
@@ -57,6 +61,7 @@ public void run() {
                     T result = task.call();
```
Comment (giftkugel) [inline platform/dolphin-platform-remoting-server/src/main/java/com/canoo/dp/impl/server/context/ClientSessionExecutorImpl.java]: Add `final` keyword.

**Comment by blackdrag** [comment]
thx

**Pull request merged**